lib/blacklight/xml_parser.rb

```diff
@@ -113,7 +113,8 @@ def self.get_master_parent(pre_data, parents_ids, parent_id)
   # non-metadata file.
   ##
   def self.iterate_files(zipfile)
-    files = zipfile.glob("*/**/**")
+    files = zipfile.entries.select(&:file?)
```

Comment:
If we're always selecting all files (via `**/**`) then we might as well just use `entries`. It's simpler, clearer, and easier to understand at a glance. If we ever need to match a different pattern, we can refactor that back to `glob`.